docs/quickstart.md

```diff
@@ -25,44 +25,50 @@ commands need to be run by a user that is in the `docker` group or by the
 `root` user.
 
 Depending on your Engine system configuration, you may be required
-to preface each `docker` command with `sudo`. One way to avoid having to use
-`sudo` with the `docker` commands is to create a Unix group called `docker` and
-add users that will be entering `docker` commands to the 'docker' group.
+to preface each `docker` command with `sudo`. If you want to run without using
+`sudo` with the `docker` commands, then create a Unix group called `docker` and
+add the user to the 'docker' group.
 
 For more information about installing Docker Engine or `sudo` configuration, refer to
 the [installation](installation/index.md) instructions for your operating system.
 
 
 ## Download a pre-built image
+To pull an `ubuntu` image, run:
 
     # Download an ubuntu image
     $ docker pull ubuntu
 
-This will find the `ubuntu` image by name on
-[*Docker Hub*](userguide/containers/dockerrepos.md#searching-for-images)
+This downloads the `ubuntu` image by name from [Docker Hub](https://hub.docker.com) to a local
+image cache. To search for an image, run `docker search`. For more information, go to:
+[*Searching images](userguide/containers/dockerrepos.md#searching-for-images)
 and download it from [Docker Hub](https://hub.docker.com) to a local
 image cache.
 
 > **Note**:
 > When the image is successfully downloaded, you see a 12 character
 > hash `539c0211cd76: Download complete` which is the
-> short form of the image ID. These short image IDs are the first 12
-> characters of the full image ID - which can be found using
+> short form of the Image ID. These short Image IDs are the first 12
+> characters of the full image ID. To view this information, run
 > `docker inspect` or `docker images --no-trunc=true`.
 
+To display a list of downloaded images, run `docker images`.
+
 ## Running an interactive shell
 
 To run an interactive shell in the Ubuntu image:
 
     $ docker run -i -t ubuntu /bin/bash       
 
-The `-i` flag starts an interactive container. The `-t` flag creates a
-pseudo-TTY that attaches `stdin` and `stdout`.  
+The `-i` flag starts an interactive container.
+The `-t` flag creates a pseudo-TTY that attaches `stdin` and `stdout`.  
+The image is `ubuntu`.
+The command `/bin/bash` starts a shell you can log in.
 
 To detach the `tty` without exiting the shell, use the escape sequence
-`Ctrl-p` + `Ctrl-q`. The container will continue to exist in a stopped state
-once exited. To list all containers, stopped and running, use the `docker ps -a`
-command.
+`Ctrl-p` + `Ctrl-q`. The container continues to exist in a stopped state
+once exited. To list all running containers, run `docker ps`. To list all containers,
+run `docker ps -a`.
```

Comment:
Wonder if we should keep "stopped and running" here, to better clarify the difference (for beginning users)